installer/msi/build-msi.ps1

```diff
@@ -6,40 +6,113 @@
     .
 .PARAMETER Arch
     The architecture, x86 or x64.
+.PARAMETER Version
+    The version of the driver.
+.PARAMETER VersionLabel
+    The version label of the driver.
 #>
 Param(
-  [string]$Arch,
-  [string]$Version,
-  [string]$VersionLabel,
-  [string]$UpgradeCode
+    [string]$Arch,
+    [string]$Version,
+    [string]$VersionLabel
+    # [string]$UpgradeCode
 )
 
 $ErrorActionPreference = 'Stop'
 
 $ProjectName = "Atlas SQL ODBC"
-$sourceDir = pwd
-$resourceDir = pwd
-$binDir = pwd
+$sourceDir = Get-Location
+$resourceDir = Get-Location
+$binDir = Get-Location
```

Comment:
On windows, `pwd` is an alias for `Get-Location`. Since this is a powershell script, it's better to use `Get-Location`

Reply:
Still disagree. `Get-Location` is super long and less clear what it is. Everyone knows `pwd`.